ext/vscode/src/commands/devCenterMode.ts

```diff
@@ -6,22 +6,25 @@ import * as vscode from 'vscode';
 import { createAzureDevCli } from '../utils/azureDevCli';
 import { execAsync } from '../utils/process';
 
-export function enableDevCenterMode(context: IActionContext): Promise<void> {
-    return setDevCenterMode(context, true);
-}
+export async function enableDevCenterMode(context: IActionContext): Promise<void> {
+    const azureCli = await createAzureDevCli(context);
+    const command = azureCli.commandBuilder
+        .withArg('config')
+        .withArg('set')
+        .withArg('platform.type')
+        .withArg('devcenter');
 
-export async function disableDevCenterMode(context: IActionContext): Promise<void> {
-    return setDevCenterMode(context, false);
+    await execAsync(command.build(), azureCli.spawnOptions());
+    void vscode.window.showInformationMessage(vscode.l10n.t('Azure Developer CLI\'s Dev Center mode has been enabled.'));
 }
 
-async function setDevCenterMode(context: IActionContext, enable: boolean): Promise<void> {
+export async function disableDevCenterMode(context: IActionContext): Promise<void> {
     const azureCli = await createAzureDevCli(context);
     const command = azureCli.commandBuilder
         .withArg('config')
-        .withArg('set')
-        .withArg('devCenterMode')
-        .withArg(enable ? 'on' : 'off');
+        .withArg('unset')
+        .withArg('platform.type');
 
     await execAsync(command.build(), azureCli.spawnOptions());
-    void vscode.window.showInformationMessage(vscode.l10n.t('Azure Developer CLI\'s Dev Center mode has been {0}.', enable ? vscode.l10n.t('enabled') : vscode.l10n.t('disabled')));
+    void vscode.window.showInformationMessage(vscode.l10n.t('Azure Developer CLI\'s Dev Center mode has been disabled.'));

```

Comment:
@savannahostrowski still happy with this string?

Reply:
LGTM!